### PR TITLE
Feature/licenseplate pipeline

### PR DESCRIPTION
**Pull request by Javatar81**
Merge pipeline demo into main

### CODE REVIEW

File: demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+apiVersion: 1.0.0
+metadata:
+  name: crw_ai_cpu_dxkf
+projects:
+  - name: dev_demos
+    source:
+      location: 'https://github.com/sa-mw-dach/dev_demos.git'
+      type: git
+      branch: feature
```
Comment (manhah) [inline demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml]: This shall be `main` then, right?

File: demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+apiVersion: 1.0.0
+metadata:
+  name: crw_ai_cpu_dxkf
```
Comment (manhah) [inline demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml]: Fixed name or rather name pattern?

File: demos/03_license_plate_pipe/pipeline/github-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: github-secret
+data:
+  secretToken: MTIzNDU2Nw==
```
Comment (manhah) [inline demos/03_license_plate_pipe/pipeline/github-secret.yaml:6]: This shall be a dummy password.

File: demos/03_license_plate_pipe/pipeline/licenseplate-pipeline-ext.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          value: 'true'
+        - name: verbose
+          value: 'true'
+        - name: gitInitImage
+          value: >-
+            registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:da1aedf0b17f2b9dd2a46edc93ff1c0582989414b902a28cd79bad8a035c9ea4
+        - name: userHome
+          value: /tekton/home
+        - name: revision
+          value: feature/licenseplate-pipeline
```
Comment (manhah) [inline demos/03_license_plate_pipe/pipeline/licenseplate-pipeline-ext.yaml]: main

File: demos/03_license_plate_pipe/pipeline/licenseplate-pipeline.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+          value: 'true'
+        - name: verbose
+          value: 'true'
+        - name: gitInitImage
+          value: >-
+            registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:da1aedf0b17f2b9dd2a46edc93ff1c0582989414b902a28cd79bad8a035c9ea4
+        - name: userHome
+          value: /tekton/home
+        - name: revision
+          value: feature/licenseplate-pipeline
```
Comment (manhah) [inline demos/03_license_plate_pipe/pipeline/licenseplate-pipeline.yaml]: main

File: demos/03_license_plate_pipe/setup/README.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+#Steps to setup3
```
Comment (manhah) [inline demos/03_license_plate_pipe/setup/README.md]: typo

File: demos/03_license_plate_pipe/setup/README.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+oc new-project pipelines-demo
+oc apply -f setup/workspace-pvc.yaml
+oc apply -f pipeline/licenseplate-pipeline.yaml
```
Comment (manhah) [inline demos/03_license_plate_pipe/setup/README.md]: Perhaps you could write some words about the PVC etc. Just quickly describe how to execute the demo.

**Review by manhah — CHANGES_REQUESTED** [review]

File: demos/03_license_plate_pipe/pipeline/github-secret.yaml (hunk 3 above)
Comment (Javatar81) [reply demos/03_license_plate_pipe/pipeline/github-secret.yaml:6]: It is a dummy password 1234567

File: demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml (hunk 1 above)
Comment (Javatar81) [reply demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml]: Changed

File: demos/03_license_plate_pipe/pipeline/licenseplate-pipeline.yaml (hunk 5 above)
Comment (Javatar81) [reply demos/03_license_plate_pipe/pipeline/licenseplate-pipeline.yaml]: Done

File: demos/03_license_plate_pipe/pipeline/licenseplate-pipeline-ext.yaml (hunk 4 above)
Comment (Javatar81) [reply demos/03_license_plate_pipe/pipeline/licenseplate-pipeline-ext.yaml]: Done

File: demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml (hunk 2 above)
Comment (Javatar81) [reply demos/03_license_plate_pipe/devfile/crw_ai_cpu_pipeline.yaml]:
```suggestion
  name: crw_ai_cpu_pipeline
```

**Comment by Javatar81** [comment]
Addressed all issues. Please review.

**Review by manhah — APPROVED** [review]
Even if it's a little late: Perfect, thanks! :-)

**Pull request merged**